site/docs/v1/tech/apis/identity-providers/_samlv2-request-body.adoc

```diff
@@ -101,9 +121,15 @@ ifdef::samlv2_idp_initiated[]
 This field must be set to `SAMLv2IdPInitiated`.
 endif::[]
 
+[field]#identityProvider.uniqueIdClaim# [type]#[String]# [optional]#Optional#::
+The name of the unique `userId` claim (attribute in the `Assertion` element) in the SAML response that FusionAuth uses to uniquely link the user. If this is not set, the `emailClaim` will be used.
+
 [field]#identityProvider.useNameIdForEmail# [type]#[Boolean]# [optional]#Optional# [default]#Defaults to `false`#::
 Whether or not FusionAuth will use the `NameID` element value as the email address of the user for reconciliation processing. If this is false, then the `emailClaim` property must be set.
 
+[field]#identityProvider.usernameClaim# [type]#[String]# [optional]#Optional#::
+The name of the claim in the SAML response (attribute in the `Assertion` element) in the SAML response that FusionAuth uses to identity the username. If this is not set, the `NameId` value will be used to link a user. This value is required if linking on username.
```

Comment:
I don't think we should describe a claim as an attribute. An attribute is meta data on an XML element.